src/vivarium_csu_ltbi/tools/number_and_percent_under5_with_hh_AcTB.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd, numpy as np
+from db_queries import get_population
+
+def mergeCountry(df1, df2):
+    if "random_seed" in df1.columns:
+        del df1['random_seed']
+    if "random_seed" in df2.columns:
+        del df2['random_seed']
+    df = pd.merge(df1.reset_index(), df2.reset_index(), on = ["input_draw_number", "random_seed", "ltbi_treatment_scale_up.scenario"], how = 'outer')

```

Comment:
Let's change this line together.  I still have a different idea on how to combine df1 and df2.